doc/source/user_guide/metadata_structure.rst

```diff
@@ -0,0 +1,361 @@
+        │   ├── wavelength (nm)
+        │   ├── power (mW)
+        │   ├── objective_magnification
+        │   ├── Filter
+        │   │   ├── filter_type
+        │   │   ├── position
+        │   │   ├── optical_density
+        │   │   ├── cut_on_wavelength (nm)
+        │   │   └── cut_off_wavelength (nm)
+        │   └── # for SEM/TEM see HyperSpy
+        ├── Spectrometer
+        │   ├── model
+        │   ├── acquisition_mode
+        │   ├── entrance_slit_width (mm)
+        │   ├── exit_slit_width (mm)
+        │   ├── central_wavelength (nm)
+        │   ├── start_wavelength (nm)
+        │   ├── step_size (nm)
+        │   ├── Grating
+        │   │   ├── groove_density (grooves/mm)
+        │   │   ├── blazing_angle (º)
+        │   │   └── blazing_wavelength (nm)
+        │   └── Filter
+        │       ├── filter_type
+        │       ├── position
+        │       ├── optical_density
+        │       ├── cut_on_wavelength (nm)
+        │       └── cut_off_wavelength (nm)
+        ├── Detector
+        │   ├── detector_type
+        │   ├── model
+        │   ├── frames
+        │   ├── integration_time (s)
+        │   ├── saturation_fraction
+        │   ├── binning
+        │   ├── processing
+        │   ├── sensor_roi
+        │   └── pixel_width (µm)
+        └── Spectral_image
+            ├── mode
+            ├── step_size
+            ├── step_size_units (nm)
+            ├── drift_correction_periodicity
+            └── drift_correction_units (s)
+
+
+General
+=======
+
+See `HyperSpy-Metadata-General
+<https://hyperspy.org/hyperspy-doc/current/user_guide/metadata_structure.html#general>`_.
+
+Sample
+======
+
+See `HyperSpy-Metadata-Sample
+<https://hyperspy.org/hyperspy-doc/current/user_guide/metadata_structure.html#sample>`_.
+
+Signal
+======
+
+signal_type
+    type: string
+
+    String that describes the type of signal. The LumiSpy specific signal classes are
+    summarized under :ref:`signal_types`.
+
+quantity
+    type: string
+
+    The name of the quantity of the “intensity axis” with the units in round brackets if
+    required, for example 'Intensity (counts/s)'.
+
+See `HyperSpy-Metadata-Signal
+<https://hyperspy.org/hyperspy-doc/current/user_guide/metadata_structure.html#sample>`_
+for additional fields.
+
+Acquisition Instrument
+======================
+
+Laser / SEM / TEM
+=================
+
+For **SEM** or **TEM** see `HyperSpy-Metadata-SEM
+<https://hyperspy.org/hyperspy-doc/current/user_guide/metadata_structure.html#sem>`_
+or `HyperSpy-Metadata-TEM
+<https://hyperspy.org/hyperspy-doc/current/user_guide/metadata_structure.html#tem>`_.
+
+
+Laser
+-----
+
+laser_type
+    type: string
+
+    The type of laser used, e.g. 'HeCd'.
+
+model
+    type: string
+
+    Model of the laser (branding by manufacturer).
+
+wavelength
+    type: float
+
+    Emission wavelength of the exciting laser in nm.
+
+power
+    type: float
+
+    Measured power of the excitation laser in mW.
+
+objective_magnification
```

Comment:
Would `magnification` on its own be enough?

Reply:
Well, the laser itself is not magnified, that is why I came up with this additional specification. On the other hand, SEM/TEM do have a field `magnification` so it could make sense to use the same here.